collection.md

```diff
@@ -0,0 +1,40 @@
+# Community automation collection
+
+## How to use collections
+
+Create or update your `requirements.yml` file found in your play folder. 
+
+Add the `community-automation.git` as shown below:
+
+```
+---
+collections:
+  - name: community.kubernetes
+    version: 1.1.1
+  - name: community.okd
+    version: 1.1.0
+  - name: https://github.com/IBM/community-automation.git
+    type: git
+```
+
+Install this by invoking the following command:
+```
+ansible-galaxy install -r requirements.yml
+```
+
+Check if this was installed using the following command:
+```
+ansible-galaxy collection list
+```
+
+## Using in your playbook
+
+```
+---
+- name: Prepare cluster for Demo Deployment
+  hosts: localhost
+  gather_facts: no
+  tasks:
```

Comment:
Should we switch this to roles, rather than encouraging include_role: too much.

Reply:
I passed this question on to Rahul

Reply:
Sure we can do that as well. The reason I showed that `include_role` was to showcase on how a task can also be included.

Reply:
added "roles:" example